### PR TITLE
fix: color of the navigation bar for the ChangeHandleViewController IC-56

### PR DESCRIPTION
**Pull request by agisilaos**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/IC-56" title="IC-56" target="_blank">TaskIC-56</a>  The color of the ChangeHandleViewController is transparent instead of black
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we fix the color of the navigation bar from transparent to black.

| BEFORE  | AFTER  |
|---|---|
| Screenshot 2022-06-03 at 10 53 06  |  Screenshot 2022-06-03 at 13 31 20  |
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by BrzozowskiPawel** [comment]
Code looks good to me 👍

File: Wire-iOS/Sources/UserInterface/Settings/SettingsStyleNavigationController.swift [hunk 1]
```diff
@@ -18,19 +18,20 @@
...
 import UIKit
+import WireCommonComponents
 
 final class SettingsStyleNavigationController: UINavigationController {
     override func viewDidLoad() {
         super.viewDidLoad()
-
         self.navigationBar.setBackgroundImage(UIImage(color: .black, andSize: CGSize(width: 1, height: 1)), for: .default)
         self.navigationBar.isTranslucent = false
+        self.navigationBar.backgroundColor = .black
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Settings/SettingsStyleNavigationController.swift:28]: What about light mode?

Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Settings/SettingsStyleNavigationController.swift:28]: There isn't a light mode for that Navigation controller. Only appears when initially want to choose your own handle.

Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/Settings/SettingsStyleNavigationController.swift:28]: I mean is that setting the color to black regardless of the application mode (light/dark) doesn't look correct. Even if the user does not see any changes in light mode, I think it's better to use the background color appreciated for the selected mode.

Comment (johnxnguyen) [reply Wire-iOS/Sources/UserInterface/Settings/SettingsStyleNavigationController.swift:28]: I agree, even if now it it's only in dark mode, doesn't mean it always will be.

**Pull request closed without merging**